### DISCUSSION TITLE
Very often initialization

### DISCUSSION THREAD

**Discussion by Robertarduino** [ClassicDIY/SkyeTracker#35] [Q&A] [answered]
I observed after some time of system working that each 30min system is making self initialization. I already change power supply. Add stabilization for power unit but not help. Do you find such behavior? 
I also disconnect unit from WiFi to not lost power or some communications problems. Buta all this action Not help.  Any advice?

**Comment by graham22** [comment]
You can monitor the log output using VSCode, the Arduino IDE or CoolTerm on a PC over the USB COM port. It should log the reason for a restart...
I would recommend powering the actuators from a separate source or see if it resets after 30 minutes without the actuators connected to help determine if it's a power issue.

**Reply by Robertarduino** [reply]
I already have two power supply to prevent ESP unit from power failure issue. I start with CoolTerm capture. First observation:  Each connection to COM port cause reset the ESP32. But after connection to COM in CoolTerm I dont see any data connected with request to move or or change state for ESP32. Example of report: 
rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:12784
load:0x40080400,len:3032
entry 0x400805e4
Config size: 302
Wrong config version.
AP password was not set in configuration
State changing from: 0 to 1
Setting up AP: SkyeTracker
With default password: <hidden>
AP IP address: 192.168.4.1
State changed from: 0 to 1
Connection to AP.
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1
ets Jun  8 2016 00:22:57

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:12784
load:0x40080400,len:3032
entry 0x400805e4
Config size: 302
Wrong config version.
AP password was not set in configuration
State changing from: 0 to 1
Setting up AP: SkyeTracker
With default password: <hidden>
AP IP address: 192.168.4.1
State changed from: 0 to 1
Connection to AP.
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for proinfo.pandasoftware.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for proinfo.pandasoftware.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1
[E][WebServer.cpp:617] _handleRequest(): request handler not found
Request for www.msftconnecttest.com redirected to 192.168.4.1

I will try to make longer connection observation with log saving. If you have other idea please let me know. Thank you.

**Comment by Robertarduino** [comment, marked as answer]
Changing power supply for bigger one resolved issue.
